sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -202,31 +216,42 @@ def _generate_select_expression(
                                     )
                                     FROM UNNEST({'.'.join(path + [node_name])}) AS {node_name}
                                 ) AS {node_name}
-                            """)
+                            """
```

Comment:
Is it Black making these formatting changes?  I'd think the closing quotes should be aligned at the same level of indentation as the opening quotes.
```suggestion
                                """
```
(ditto for the similar multi-line strings later in the file)